src/lbfgs.jl

```diff
@@ -92,6 +92,7 @@ LBFGSOperator{T}(
 has_args5(op::LBFGSOperator) = true
 use_prod5!(op::LBFGSOperator) = true
 isallocated5(op::LBFGSOperator) = true
+storage_type(op::LBFGSOperator{T}) where {T} = Vector{T}
```

Comment:
LBFGS and LSR1 are coded explicitly with CPU vectors, because they access somes indices directly. Maybe they could be defined on the GPU but this should be done in another PR because it would required a lot of modifications.

Reply:
Ok, that makes sense.